internal/pkg/tfconf/module.go

```diff
@@ -120,20 +123,53 @@ func CreateModule(path string) (*Module, error) {
 		}
 	}
 
+	type terraformOutput struct {
```

Comment:
I'm going to turn this into a public struct and move it into module.go so that the code stays pretty